ksqldb-engine/src/main/java/io/confluent/ksql/structured/SchemaKSourceFactory.java

```diff
@@ -237,4 +243,10 @@ private static LogicalSchema resolveSchema(
         buildContext.getFunctionRegistry()
     ).resolve(step, dataSource.getSchema());
   }
+
+  private static int getPseudoColumnVersionFromConfig(final KsqlConfig ksqlConfig) {
```

Comment:
nit: use the public version in SystemColumns, rather than duplicating the code here.